auth-web/tests/unit/views/HomeView.spec.ts

```diff
@@ -131,20 +126,4 @@ describe('HomeView.vue', () => {
     expect(createAccountLink).toBeDefined()
     expect(createAccountLink.textContent).toContain('Create a BC Registries Account')
   })
-
-  it('renders the LoginBCSC when the create account dialog is called', async () => {
```

Comment:
any reason why this is getting removed? if the behaviour is changing, do we need another test?

Reply:
This case was no longer valid, as the popup for the extra provincial is removed